libraries/Microsoft.Bot.Builder/Teams/TeamsInfo.cs

```diff
@@ -378,6 +378,23 @@ public static async Task<string> SendMessageToAllUsersInTenantAsync(ITurnContext
             }
         }
 
+        /// <summary>
+        /// Cancels the proccess of an operation.
```

Comment:
Be careful with the typos. Check everything before creating the PRs.
```suggestion
        /// Cancels the process of an operation.
```